services/scanner.py

```
@@ -384,128 +296,100 @@ def add_sys(sys_id, sys_type, name, conf, ev_type, ev_val):
+            # 4. Check Meta tags
+            if not matched and 'meta' in app_spec and metas:
+                for m_name, m_regex in app_spec['meta'].items():
+                    m_val = metas.get(m_name)
+                    if m_val and re.search(m_regex.split(';')[0], m_val, re.IGNORECASE):
+                        add_sys(app_name, app_spec, 0.95, 'meta', m_name)
+                        matched = True
+                        break
 
+        # Process implies logic recursively
+        new_additions = {}
+        for app_name, sys in systems_map.items():
```

Ruff (0.15.14)

[warning] Loop control variable `sys` not used within loop body

Rename unused `sys` to `_sys`

(B007)